frontend/login.html

```diff
@@ -10,21 +10,28 @@
     </head>
     <body>
         <h1>Mars Rover Project</h1>
-        <h2>Login</h2>
-        <p>Login here to access your saved NASA images and data.</p>
-        <form class="login" action="/authenticate" method="POST">
+        <div class="login">
+            <h2>Login to access your NASA images and data</h2>
+            <form action="/authenticate" method="POST">
+                <div class="form-input">
+                    <label for="username">Username</label>
+                    <input type="text" id="username" name="username">
+                </div>
+                <div class="form-input">
+                    <label for="password">Password</label>
+                    <input type="password" id="password" name="password">
+                </div>
+                <div class="form-input">
+                    <input type="submit" name="" value="Login" >
```

Comment:
I don't know if we need a name for the submit button

Reply:
Ok Will remove.

Reply:
Done